### DISCUSSION TITLE
Mounting host drive

### DISCUSSION THREAD

**Discussion by pgmillon** [cockpit-project/cockpit-machines#746] [Q&A]
Hi,
this is probably a dumb question since I'm not too familiar with qemu/cockpit: is it possible to attach a SATA device to a VM ?
I already successfully attached a USD drive via the Host devices panel but I'm looking to attach /dev/sda & /dev/sdc and I'm not sure how to do that using cockpit-machines.

```
$ sudo lsscsi
[0:0:0:0]    disk    ATA      ST1000DM010-2EP1 CC46  /dev/sda
[1:0:0:0]    disk    ATA      INTEL SSDSC2BW12 DC32  /dev/sdb
[2:0:0:0]    disk    ATA      ST1000DM010-2EP1 CC46  /dev/sdc
```

**Comment by skobyda** [comment]
Hey.
Libvirt offer possibility to attach such device as "block" , however that is not exposed thru the Cockpit UI yet.
To achieve this from command line, you can try something like:
`virsh attach-disk VM_NAME /dev/sda sda --config --targetbus sata --sourcetype block`
